test/test_synth_chorus_reverb.c

```diff
@@ -0,0 +1,39 @@
+
+#include "test.h"
+#include "fluidsynth.h"
+
+// this test should make sure that sample rate changed are handled correctly
+int main(void)
+{
+    fluid_synth_t *synth;
+    fluid_settings_t *settings = new_fluid_settings();
+    TEST_ASSERT(settings != NULL);
+    
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.reverb.roomsize", 1.0));
+    TEST_SUCCESS(fluid_settings_setnum(settings, "synth.reverb.damp", 1.0));
```

Comment:
If you would use different values for the parameters, the tests could actually check that the correct value is returned/set for a named parameter.

Reply:
> the tests could actually check that the correct value is returned/set for a named parameter.

This is done below by asking the synth to return these values (which he must have read from the settings object to be correct). For the correct saving of the settings, relying on the return value of the settings API is sufficient here.

Reply:
Sorry, I mean by settings damp to 1.0, roomsize to 0.9, etc we could test that ```fluid_synth_get_reverb_damp``` does actually return the value set by synth.reverb.damp, and not the one set via synth.reverb.roomsize.